collective/trustedimports/soap.rst

```diff
@@ -0,0 +1,61 @@
+SOAP
+===
+
+When used in RestrictedPython we can still use Zeep
+
+>>> teval("from zeep import Client;return Client('http://webservices.oorsprong.org/websamples.countryinfo/CountryInfoService.wso?WSDL')")
+<zeep.client.Client ... at ...>
+
+>>> teval("from zeep.exceptions import Error;raise Error('Test error')")
+Traceback (most recent call last):
+...
+Error: Test error
+
+
+We cann't set transport option for WSDL service
```

Comment:
The code below doesn't show setting a transport

Reply:
@djay , add test for transport